_episodes/03-create.md

```diff
@@ -143,6 +143,16 @@ wording of the output might be slightly different.
 >
 > > ## Solution -- USE WITH CAUTION!
 > >
+> > ### Background
+> > Removing files from a git repository needs to be done with caution. To remove files from the working tree and not from your working directory, use
+> > ~~~
+> > $ rm filename
+> > {: .language-bash}
+> > 
+> > The file being removed has to be in sync with the branch head with no updates. If there are updates, the file can be removed by force by using the `-f` option. Similarly a directory can be removed from git using `rm -r dirname` or `rm -rf dirname`.
```

Comment:
This could also be addressed in #588: Are you talking about `git rm` here? I'm very confused by this explanation, because neither `rm` nor `git rm` actually removes a file from the repository (sticking to the terminology in https://git-scm.com/about/staging-area).